lint.sh

```diff
@@ -29,8 +36,14 @@ docker run --rm -v "$(pwd)":/code -v "$(pwd)/.hadolint.yml":/.hadolint.yml hadol
 # https://github.com/koalaman/shellcheck#from-your-terminal
 find . -type d -name node_modules -prune -false -o -name "*.sh" -print0 | xargs -0 docker run --rm -v "$(pwd)":/code dcycle/shell-lint:2
 
+if ! [[ -f "local.env" ]]; then
+	touch local.env
```

Comment:
If `local.env` doesn't exist Docker will complain when the test-runner is ran (which is used for linting as well as test runs).